src/MOI_wrapper.jl

```diff
@@ -603,43 +603,51 @@ end
 # ==============================================================================
 #    Callbacks in Gurobi
 # ==============================================================================
-struct CallbackFunction <: MOI.AbstractOptimizerAttribute end
-function MOI.set(model::Optimizer, ::CallbackFunction, f::Function)
-    set_callback_func!(model.inner, f)
+
+struct CallbackFunction <: MOI.AbstractModelAttribute end
+
+function MOI.set(model::Optimizer, ::CallbackFunction, callback::Function)
+    set_callback_func!(model.inner, callback)
     update_model!(model.inner)
     return
 end
 
 """
-    loadcbsolution!(m::Optimizer, cb_data::GurobiCallbackData, cb_where::Int)
+    cbget_mipsol_sol(m::Optimizer, cb_data::GurobiCallbackData, cb_where::Int)
 
 Load the variable primal solution in a callback.
 
 This can only be called in a callback from `CB_MIPSOL`. After it is called, you
 can access the `VariablePrimal` attribute as usual.
 """
-function loadcbsolution!(model::Optimizer, cb_data::CallbackData, cb_where::Cint)
+function cbget_mipsol_sol(model::Optimizer, cb_data::CallbackData, cb_where::Cint)
     if cb_where != CB_MIPSOL
         error("loadcbsolution! can only be called from CB_MIPSOL.")
     end
     Gurobi.cbget_mipsol_sol(cb_data, cb_where, model.variable_primal_solution)
+    model.termination_status = MOI.SOLUTION_LIMIT
```

Comment:
@mlubin, @blegat What is the termination status at an integer solution in a callback?